src/page-multi-facility/CreateBaselineScenarioPage.tsx

```diff
@@ -33,11 +34,29 @@ const ModalContent = styled.div`
 `;
 
 const CreateBaselineScenarioPage: React.FC = () => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
I tried to add this lint diabler, but as you can see it didn't work.  Not really sure what to do about this

Reply:
I had tried the same thing on a no-unused-vars lint failure elsewhere about a week ago or so and also couldn't get it to calm down. I say leave it for now instead of spending time on it.